Add notice about stopping support for v1.21 (#1659)
aws/eks-distro: README.md

| --- | --- | --- |
| 24 | [v1-21-eks-24](https://distro.eks.amazonaws.com/kubernetes-1-21/kubernetes-1-21-eks-24.yaml) | [v1.21.14](https://github.com/kubernetes/kubernetes/release/tag/v1.21.14) |

**🚨 IMPORTANT INFORMATION ABOUT v1.21 SUPPORT 🚨**

> Aligning with the [Amazon EKS release calendar](https://docs.aws.amazon.com/eks/latest/userguide/kubernetes-versions.html#kubernetes-release-calendar), 

Reviewer: ```suggestion
In alignment with the [Amazon EKS release calendar](https://docs.aws.amazon.com/eks/latest/userguide/kubernetes-versions.html#kubernetes-release-calendar), 
```

Reviewer: To ensure it doesn't sound like we're just doing this.

EKS Distro will be discontinuing support of Kubernetes v1.21 soon. 

> While there are no plans to removed EKS-D v1.21 images from the [ECR](https://gallery.ecr.aws/eks-distro),

Reviewer: ```suggestion
While there are no plans to remove EKS-D v1.21 images from the [ECR](https://gallery.ecr.aws/eks-distro),
```

Author: added

more updates, including security fixes, for them.

**Due to the increased security risk this poses, it is HIGHLY recommended that

> users of v1.18 - v1.20 update to a supported version (v1.22+) as soon as

Reviewer: ```suggestion
users of v1.18 - v1.21 update to a supported version (v1.22+) as soon as
```

Reviewer: Should this be 1.21 instead?
